android/app/src/main/java/com/github/quarck/calnotify/dismissedeventsstorage/DismissedEventAlertRecord.kt

```diff
@@ -36,11 +37,11 @@ enum class EventDismissType(val code: Int) {
         get() = true; // this != EventMovedUsingApp
 
     val canBeRestored: Boolean
-        get() = this != AutoDismissedDueToCalendarMove && this != EventMovedUsingApp
+        get() = this != AutoDismissedDueToCalendarMove && this != EventMovedUsingApp && this != AutoDismissedDueToRescheduleConfirmation
```

Comment:
lets investigate this canBeRestored flag deeper. I think its that we can restore when something is manaullly dissmised in a way that you couldn't if the app did the EventMovedUsingApp flow.


but we might with reschedule confirmations

but actaully a lot of the reasons we need this is the event is not associated with an event in the calendar db on the phone so maybe not

Reply:
technically some of the events we dismiss this way can be restored. but not all (like if we changed phones)

so still kinda on the fence about which to put here.

but it turns out nothing else in the code really uses the property so prolly doesnt matter that much